lib/rubocop/cop/style/ternary_parentheses.rb

```diff
@@ -86,10 +88,14 @@ def only_closing_parenthesis_is_last_line?(condition)
           condition.source.split("\n").last == ')'
         end
 
-        def condition_as_parenthesized_one_line_pattern_matching?(condition)
+        def condition_as_parenthesized_csend_operator_or_one_line_pattern_matching?(condition)
           return false unless condition.parenthesized_call?
           return false unless (first_child = condition.children.first)
 
+          if first_child.csend_type? && first_child.operator_method? && !first_child.parenthesized?
+            return true
+          end
```

Comment:
The same thing would happen without safe nav btw:

```ruby
# test.rb
(foo.> bar) ? 1000 : 2000

$ rubocop -A test.rb --only Style/TernaryParentheses
Inspecting 1 file
C

Offenses:

test.rb:1:1: C: [Corrected] Style/TernaryParentheses: Omit parentheses for ternary conditions.
(foo.> bar) ? 1000 : 2000
^^^^^^^^^^^^^^^^^^^^^^^^^

# test.rb becomes
foo.> bar ? 1000 : 2000
```

Reply:
Good catch! I've updated this PR.